public/search.js

```diff
@@ -1,7 +1,7 @@
 var search_box = document.querySelector('#nav_search_box');
 search_box.addEventListener('change', search);
 function search(event){
-    var url='/search/'+search_box.value.replace(/\s+/g, '').toLowerCase();
+    var url='/search/'+search_box.value.replace(/\s+/g, '+');
```

Comment:
I like. Why not to lowercase? Just no need? Do you think the link would look better with lowercase?

Reply:
I felt like case-sensitive searches could be useful, but if not I can just tack that back on.

Reply:
nvm its not case-sensitive because of db index searching now